src/formatting.tsx

```diff
@@ -302,5 +302,5 @@ export function formatAddress (address?: IAddress | null) {
 }
 
 export function formatAddressMultiline (address?: IAddress | null) {
-  return parser(formatAddress(address).replace(/, /g, '<br/>'));
```

Comment:
what does /g do? Also I thought .replace only took two params?

Reply:
Ah, I see why this is confusing. replace takes 2 params. The first can either be a string or a 'regex' pattern. The pattern begins and ends in a `/`. You can add a g after the final slash to make it a 'global' pattern. `', '` will only replace the first `, ` whereas `/, /g` will replace ALL `, `.

Since now I just want it to do 2 lines not 3, I removed the `g`. However, I could have just switched it to a string to do the same thing. Updated PR. Thanks for the great comment!